Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.23-bullseye
+FROM golang:1.23
 
 WORKDIR /app
```

Comment:
When you WORKDIR command the docker context automaticaly cd's to this directory.